### PR TITLE
Term: Updates the global styles: color palette, typography

### PR DESCRIPTION
**Pull request by mlaetitia**
Changes the color palette to include the colors in Figma and typography:
- Heading font sizes
- Font sizes for small, regular, ...
- Default capitalization
- Default font

### CODE REVIEW

File: term/theme.json [hunk 1]
```diff
@@ -18,16 +37,95 @@
...
+			"h1": {
+				"typography": {
+					"fontSize": "64px",
+					"fontStyle": "normal",
+					"fontWeight": "700"
+				}
+			},
+			"h2": {
+				"typography": {
+					"fontSize": "48px"
```
Comment (MaggieCabrera) [inline term/theme.json]: can any of these use preset values? if not, let's use rem values instead so they can scale

File: term/theme.json [hunk 2]
```diff
@@ -18,16 +37,95 @@
...
+				}
+			},
+			"link": {
+				"color": {
+					"text": "var(--wp--preset--color--custom-grey)"
+				}
+			}
+		},
 		"spacing": {
+			"blockGap": "25px",
```
Comment (MaggieCabrera) [inline term/theme.json]: rem units are better for spacing too

File: term/theme.json [hunk 3]
```diff
@@ -36,7 +134,10 @@
 			}
 		},
 		"typography": {
-			"fontFamily": "var(--wp--preset--font-family--system-font)"
+			"fontFamily": "var(--wp--preset--font-family--system-font)",
+			"fontSize": "var(--wp--preset--font-size--medium)",
+			"lineHeight": "1.5",
+			"textTransform": "capitalize"
```
Comment (MaggieCabrera) [inline term/theme.json:140]: I think the body font weight needs to be thinner (but maybe it's my mac, that doesn't have helvetica!)

Comment (mlaetitia) [reply term/theme.json:140]: On mine, I can see the font in a size size, but I can definitely change it back as it was, or not set a default. What do you think is the best way here ?

**Comment by mlaetitia** [comment]
@MaggieCabrera Thanks for the review! Addressed your comments and happy to make any needed changes!

**Review by MaggieCabrera — APPROVED** [review]
Let's bring this in and tweak it as we go

**Pull request merged**